services/bundle_analysis/notify/messages/comment.py

```diff
@@ -184,3 +197,46 @@ def _create_bundle_rows(
             )
 
         return bundle_rows
+
+    def _create_bundle_route_data(
```

Comment:
Would we use this computation/stats elsewhere in the code, like in GQL for example? In case it makes sense for this to live in Shared

Reply:
Not really, these computations are really for PR comments as it figures out the icons to use and how to display the size (eg bytes, MB, etc). On the app side these are taken care of by gazebo